index.html

```diff
@@ -20,20 +20,192 @@
             </div>
         </div>
         </div>
-    </header>
-    <main>
+
         <section>
-            <h1>Nwannah  <br>Cynthia</h1>
+            <h2>Nwannah  <br>Cynthia</h2>
             <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. <br> 
                 If you like what you see and have a project you need coded, don’t hesitate and contact me</p>
             <div class="bot">
             <input type="button" value="Start Collaboration ">
         </div>
-
     </section>
+    </header>
+    
+    <main>    
     <div class="sectiontwo">
-        <!-- <img src="./asset/shape.svg" alt="A joined shape image"> -->
+        <img src="./asset/shape.svg" alt="">
+    </div>
+    <div class="pro">
+        <h1>projects</h1>
+    </div>
+    <div id="section-three">
+        <div class="lap">
+            <h1>welcome</h1>
+        <img src="./asset/laptop-image.png" alt="">
+    </div>
+    </div>
+    <div class="section-threeb">
+        <div class="main-se">
+        <div class="one"><p>My Project Portfolio</p></div>
+        <div class="two"><button type="button">HTML/CSS</button></div>
+        <div class="three"><button type="button">Ruby on Rails</button></div>
+        <div class="four"><button type="button">Javascript</button></div>
+        <div class="five"><p>See this project <img src="./asset/arrowright.png" alt=""></p></div>
+    </div>
+    </div>
+    <div id="section-threeb">
+        <div class="lap">
+        <img src="./asset/laptop2image.png" alt="">
+    </div>
+    </div>
+    <div class="section-threeb">
+        <div class="main-se">
+        <div class="one"><p>My Project Portfolio</p></div>
+        <div class="two"><button type="button">HTML/CSS</button></div>
+        <div class="three"><button type="button">Ruby on Rails</button></div>
+        <div class="four"><button type="button">Javascript</button></div>
+        <div class="five"><p>See this project <img src="./asset/arrowright.png" alt=""></p></div>
```

Comment:
- [ ] This should be wrapped in either a `button` tag or an `a` tag since it is a button that performs an action when clicked. Please do that for all other buttons of your listed projects
- [ ] Please add the pseudo-classes to these buttons requested on the Figma template as seen in the screenshot below 👌 
![image](https://user-images.githubusercontent.com/45460192/170796661-a00ce6f1-e21e-42cf-b310-662073f83d28.png)
- [ ] The arrow icon you have here is overlapping the text in the button as seen below. Please fix that 👍 
![image](https://user-images.githubusercontent.com/45460192/170797242-b4df933c-56fe-4231-ad2b-96eb29057ca4.png)